app/controllers/items_controller.rb

```diff
@@ -536,10 +537,21 @@ def schema_validation
   end
 
   def refresh_metadata
+    if @object.catkey.blank?
+      render status: :forbidden, plain: 'object must have catkey to refresh descMetadata'
```

Comment:
yeah.  this is a case that the user really shouldn't run into if they're not manually constructing URLs, because we won't present the button for things without a catkey.  if the user did run into this, they'd get a plaintext page with the error message a 403 forbidden as the HTTP status code.